automata/pda/pda.py

```diff
@@ -2,16 +2,33 @@
 """Classes and methods for working with all pushdown automata."""
 
 import abc
+from typing import AbstractSet, Union
+
+from typing_extensions import Literal
 
 import automata.base.exceptions as exceptions
 import automata.pda.exceptions as pda_exceptions
-from automata.base.automaton import Automaton
+from automata.base.automaton import Automaton, AutomatonStateT, AutomatonTransitionsT
+from automata.pda.configuration import PDAConfiguration
+from automata.pda.stack import PDAStack
+
+PDAStateT = AutomatonStateT
+PDATransitionsT = AutomatonTransitionsT
+PDAAcceptanceModeT = Union[
```

Comment:
@caleb531 
```suggestion
PDAAcceptanceModeT = Literal["final_state", "empty_stack", "both"]
```

Reply:
@eliotwrobson You can do that?? I mean, uhh, I was going to fix that eventually lol. 😅

Ha and you can see how my inexperience with Python types is starting to show. So thanks for the pointer!